<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorClass</class>
 <widget class="QWidget" name="ScriptEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScriptEditor</string>
  </property>
  <widget class="QCodeEditor" name="codeEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>560</width>
     <height>291</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::LineWrapMode::NoWrap</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAsButton">
   <property name="geometry">
    <rect>
     <x>487</x>
     <y>310</y>
     <width>39</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save file as...</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fileNowLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>311</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>39</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh editor with file content. Warning: This will replace unsaved changes in the editor.</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>310</y>
     <width>39</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>401</x>
     <y>310</y>
     <width>39</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open file...</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCodeEditor</class>
   <extends>QTextEdit</extends>
   <header>QCodeEditor</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
